XMTP.podspec

```diff
@@ -44,5 +44,5 @@ Pod::Spec.new do |spec|
   spec.dependency "web3.swift"
   spec.dependency "GzipSwift"
   spec.dependency "Connect-Swift", "= 0.12.0"
-  spec.dependency 'LibXMTP', '= 0.5.0-beta0'
+  spec.dependency 'LibXMTP', '= 0.5.0-beta1'
```

Comment:
Do you mind bumping the pod version as well here to 0.11.0?

Reply:
Added 25bd9d16143429e7aa41e6c2c0cc77ddb1ebba06